README.md

```diff
@@ -46,6 +46,8 @@ ytimport:
   split_tracks: true
   group_albums: true
   quiet_fallback: skip # optional; alternatively, to import as is, set 'asis'.
+  oauth_client_id: ''
+  oauth_client_secret: ''
```

Comment:
Please also add/document the `cookiesfile` config option here.

Also, I think it would be good to keep all the authentication options listed together (next to `auth_headers`).

Also, would you mind briefly explaining in the README how to obtain the cookies file or link to another page that's explaining that?

(Similarly, we/I should document how to obtain the oauth client credentials but that's not blocking your PR.)